core/modules/filter/css/filter.css

```diff
@@ -91,3 +91,49 @@ input#edit-filters-filter-html-settings-allowed-html {
   display: inline-block !important;
   padding-bottom: 10px;
 }
+.image-library-choose-file .field-content {
+  font-size: 0.7rem;
+}
+.image-library-choose-file p {
+  margin: 0;
+}
+.image-library-image {
+  max-width: 150px;
+  max-height: 150px;
+}
+.library-view {
+  display:none;
+}
+.form-type-number .field-suffix {
+  display: inline;
+}
+.editor-dialog .form-item {
+  margin-bottom: 0.5em;
+}
+.editor-dialog .form-item .description {
+  margin-bottom: 0.5em;
+  margin-top: 0.2em;
+  padding: 0;
+  padding-left: 1em;
+  font-size: 0.9rem;
+}
+
+.image-library-choose-file {
+  float: left;
+  padding-left: 5px;
+  padding-right: 5px;
+  height: 170px;
+}
+
+.image-form-wrapper {
+  display: flex;
+}
+.image-form-left {
+  width:400px;
+}
+.image-form-right {
+  padding-left: 20px;
+  height: 400px;
+  overflow-y: scroll;
+  min-width: 50%;
```

Comment:
Again, curious.

Reply:
The minimum width is of course only relevant in narrower displays. I think perhaps I could better set it to be 500px rather than 50% so that the minimum image library width is three images, albeit with horizontal scrolling.